File: main/src/main/java/net/fwitz/math/main/binary/dual/functions/exponential/DualExp1OverZ.java
package net.fwitz.math.main.binary.dual.functions.exponential;

import net.fwitz.math.plot.binary.dual.DualNumberFunctionPlot;

public class DualExp1OverZ {
    public static void main(String[] args) {
        new DualNumberFunctionPlot("exp(1/z)")
                .computeFn(z -> z.inverse().exp())
                .render();
    }
}

File: main/src/main/kotlin/net/fwitz/math/main/binary/dual/functions/exponential/DualExp1OverZ.kt
package net.fwitz.math.main.binary.dual.functions.exponential

import net.fwitz.math.binary.dual.DualNumber
import net.fwitz.math.plot.binary.dual.DualNumberFunctionPlot

object DualExp1OverZ {
    @JvmStatic
    fun main(args: Array<String>) = DualNumberFunctionPlot("exp(1/z)")
        .computeFn { z: DualNumber -> z.inverse.exp }
        .render()
}